pkg/tarmak/tarmak.go

```diff
@@ -281,20 +281,21 @@ func (t *Tarmak) Version() string {
 }
 
 func (t *Tarmak) Validate() error {
-	var err error
-	var result error
+	var result *multierror.Error
```

Comment:
I think this whole method should just be calling to validateSSHSetup() or something like that

Reply:
Not too sure what you mean by this. tarmak.Validate() should only call to validateSSHSetup?

Reply:
@simonswine